packages/app/src/cli/services/dev/select-store.ts

```diff
@@ -118,6 +118,15 @@ export async function convertToTestStoreIfNeeded(
   org: Organization,
   token: string,
 ): Promise<void> {
+  /**
+   * Is not possible to convert stores to dev ones in spin environmets. Should be created directly as development.
+   */
+  if (
+    environment.service.serviceEnvironment() === environment.network.Environment.Spin &&
+    environment.local.firstPartyDev()
+  ) {
+    return
+  }
```

Comment:
All stores used in the CLI are `development stores`, what this piece of code is convert them to "test" stores, which is just development stores with transfer disabled.
The comment should reflect that.

I'll try to abstract  `environment.service.serviceEnvironment() === environment.network.Environment.Spin` into a function (maybe inside environment?), this is something that can be reused to know if we are connecting to a spin instance (independently of where the CLI is)